nldi-liquibase/src/main/resources/liquibase/characteristic_data/indexes.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset drsteini:create_index.characteristic_data.plusflowline_vaa_np21_ut
+--preconditions onFail:MARK_RAN onError:HALT
+--precondition-sql-check expectedResult:0 select count(*) from pg_class c join pg_namespace n on n.oid = c.relnamespace where n.nspname = 'characteristic_data' and c.relname = 'plusflowlinevaa_np21_ut' and c.relkind = 'i'
```

Comment:
I'm not sure which would be better. One way shows as "MARK_RAN" in the databasechangelog, the other as "EXECUTED". Of course the log can be manipulated... The precondition pattern works on our other database platform, so that is a good thing. However, I would not use it just for a false sense of portability of the code, only of the pattern.